Models/c172p.xml

```diff
@@ -388,8 +388,6 @@
     <!-- interior effects -->
     <effect>
         <inherits-from>Effects/interior/c172p-interior</inherits-from>
-        <object-name>AirVent_left</object-name>
```

Comment:
Why were these removed? Objects will now appear black when using the flash light.